__tests__/TemplateIndexer.test.js

```diff
@@ -0,0 +1,52 @@
+import { datasetFromJsonld } from 'utilities'
+import TemplateIndexer from 'TemplateIndexer'
+
+describe('Indexer', () => {
+  describe('Indexing a resource template resource template', () => {
```

Comment:
I find the construction `resource template resource template` to be confusing. Could this be adjusted as `resource template's resource template` or maybe `base resource template`?

Reply:
Clarified.